src/components/Table/Table.test.jsx

```diff
@@ -499,6 +516,23 @@ describe('Table', () => {
     expect(wrapper3.find(TableHead).prop('options').wrapCellText).toEqual('auto');
   });
 
+  it('should render RowActionsCell dropdowns in the right direction for different language directions ', async () => {
+    // Should render correctly by default even if no lang attribute exist
+    const { getByTestId, unmount, rerender } = render(
+      <Table columns={tableColumns} data={[tableData[0]]} options={options} />
+    );
+    await fireEvent.click(getByTestId('Table-row-0-row-actions-cell-overflow'));
+    expect(document.body.childNodes[2].className.includes('bx--overflow-menu--flip')).toBeTruthy();
+
+    document.documentElement.setAttribute('dir', 'rtl');
+
+    rerender(<Table columns={tableColumns} data={[tableData[1]]} options={options} />);
+    await fireEvent.click(getByTestId('Table-row-1-row-actions-cell-overflow'));
+    expect(document.body.childNodes[2].className.includes('bx--overflow-menu--flip')).toBeFalsy();
+
+    unmount();
+  });
```

Comment:
```suggestion
    const { rerender } = render(
      <Table columns={tableColumns} data={[tableData[0]]} options={options} />
    );
    await fireEvent.click(screen.getByTestId('Table-row-0-row-actions-cell-overflow'));
    expect(screen.childNodes[2].className.includes('bx--overflow-menu--flip')).toBeTruthy();

    document.documentElement.setAttribute('dir', 'rtl');

    rerender(<Table columns={tableColumns} data={[tableData[1]]} options={options} />);
    await fireEvent.click(screen.getByTestId('Table-row-1-row-actions-cell-overflow'));
    expect(screen..childNodes[2].className.includes('bx--overflow-menu--flip')).toBeFalsy();
  });
```

Reply:
I think Kent Dodds says when you are operating on the root element like I do here for the RTL that you should unmount after the test. to avoid leaking those changes into other test. I am trying to find the issue comment